mpi-proxy-split/seq_num.cpp

```diff
@@ -0,0 +1,253 @@
+#include <mpi.h>
+
+#include <map>
+#include <pthread.h>
+#include <semaphore.h>
+
+#include "jassert.h"
+#include "seq_num.h"
+#include "mpi_nextfunc.h"
+#include "virtual-ids.h"
+#include "record-replay.h"
+
+using namespace dmtcp_mpi;
+
+#define DEBUG_SEQ_NUM
```

Comment:
Quick comment: I guess this should be commented out by default.

Reply:
Yes. I will comment it out.